Improve docs: troubleshooting bad blocks, link OpenTelemetry receivers; update kube-manifests (#838)
grafana/tempo: docs/tempo/website/configuration/_index.md

## Distributor
See [here](https://github.com/grafana/tempo/blob/main/modules/distributor/config.go) for all configuration options.

> Distributors are responsible for receiving spans and forwarding them to the appropriate ingesters.

Reviewer: ```suggestion
Distributors receive spans and forward them to the appropriate ingesters.
```

> The below configuration enables all available receivers, for a production deployment you should only enable the receivers you need.
> Additional documentation about the receiver can be found in [the receiver README](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/README.md).

Reviewer: ```suggestion
The following configuration enables all available receivers. For a production deployment, enable only the receivers you need.
Additional documentation about the receiver is available in [the receiver README](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/README.md).
```
